src/ResourceManager/Network/Commands.Network/help/Get-AzureRmVirtualNetworkUsageList.md

```diff
@@ -0,0 +1,89 @@
+---
+external help file: Microsoft.Azure.Commands.Network.dll-Help.xml
+online version: 
+schema: 2.0.0
+---
+
+# Get-AzureRmVirtualNetworkUsageList
+
+## SYNOPSIS
+Gets virtual network current usage.
+
+## SYNTAX
+
+```
+Get-AzureRmVirtualNetworkUsageList -ResourceGroupName <String> -Name <String>
+```
+
+## DESCRIPTION
+The **Get-AzureRmVirtualNetworkUsageList** cmdlet gets per subnet usage for the specified virtual network.
+
+## EXAMPLES
+
+### Example 1
+```
+PS C:\> Get-AzureRmVirtualNetworkUsageList -ResourceGroupName test -Name usagetest
+
+Get-AzureRmVirtualNetworkUsageList -ResourceGroupName test -Name usagetest
```

Comment:
Updated cmdlet to return `List<PSVirtualNetworkUsage>` explicitly.
The example output sample contains list of two `PSVirtualNetworkUsage` items now